easybuild/tools/options.py

```diff
@@ -197,10 +199,12 @@ def override_options(self):
             'set-gid-bit': ("Set group ID bit on newly created directories", None, 'store_true', False),
             'sticky-bit': ("Set sticky bit on newly created directories", None, 'store_true', False),
             'skip-test-cases': ("Skip running test cases", None, 'store_true', False, 't'),
-            'umask': ("umask to use (e.g. '022'); non-user write permissions on install directories are removed",
-                      None, 'store', None),
             'optarch': ("Set architecture optimization, overriding native architecture optimizations",
                         None, 'store', None),
+            'umask': ("umask to use (e.g. '022'); non-user write permissions on install directories are removed",
+                      None, 'store', None),
+            'update-lmod-caches': ("Update Lmod cache files in specified directory", None, 'store_or_None',
```

Comment:
there is no caching in Tcl(/C) env mods... suggestion for a better name?

at least that makes it clear from the name of the command line option that this is specific to Lmod